src/nunit-gui/Presenters/MainPresenter.cs

```diff
@@ -108,12 +112,35 @@ private void InitializeMainMenu()
             _view.SaveAsCommand.Enabled = canCloseOrSave;
             _view.SaveResultsCommand.Enabled = canCloseOrSave && _model.HasResults;
             _view.ReloadTestsCommand.Enabled = canCloseOrSave;
-            _view.SelectRuntimeMenu.Enabled = canCloseOrSave;
             _view.RecentProjectsMenu.Enabled = !isTestRunning;
             _view.ExitCommand.Enabled = true;
 
+            // Special handling for available runtimes. Use model to initialize
+            // HACK: This exposes the underlying ToolStripMenuItem
+            var dropDownItems = _view.SelectRuntimeMenu.DropDownItems;
+            // Null check for when we are using a mock view to test
+            // Count check to avoid initializing twice
+            if (dropDownItems != null && dropDownItems.Count == 1)
+            {
+                foreach (var runtime in _model.AvailableRuntimes)
+                {
+                    var text = runtime.DisplayName;
+                    // Don't use Full suffix, but keep Client if present
+                    if (text.EndsWith(" - Full"))
+                        text = text.Substring(0, text.Length - 7);
+                    var menuItem = new ToolStripMenuItem(text);
+                    menuItem.Tag = runtime.ToString();
+                    dropDownItems.Add(menuItem);
+                }
+
+                _view.SelectedRuntime.Refresh();
+            }
+
             // Project Menu
             _view.ProjectMenu.Enabled = _view.ProjectMenu.Visible = _model.HasTests;
+            //_view.SelectRuntimeMenu.Enabled = canCloseOrSave;
```

Comment:
Changed my mind several times on this one. Finally decided that these three items should always be available, even during a test run, but not take effect immediately.